subsys/nrf_security/Kconfig

```diff
@@ -53,6 +53,13 @@ config PSA_PROMPTLESS
 
 if NRF_SECURITY
 
+config NRF_SECURITY_ENABLE_BUILTIN_KEYS
```

Comment:
Maybe just name it MBEDTLS_PSA_CRYPTO_BUILTIN_KEYS then?

Reply:
Sure, I'm not well versed in the realms of crypto.